gno.land/pkg/gnoweb/gnoweb.go

```diff
@@ -335,6 +335,21 @@ func handleRealmRender(logger *slog.Logger, app gotuna.App, cfg *Config, w http.
 			return
 		}
 	}
+
+	dirdata := []byte(rlmpath)
+	dirres, err := makeRequest(logger, cfg, qFileStr, dirdata)
+	if err != nil {
+		writeError(logger, w, err)
+		return
+	}
+	files := strings.Split(string(dirres.Data), "\n")
+	hasReadme := false
+	for _, v := range files {
+		if v == "README.md" {
+			hasReadme = true
+		}
+	} 
```

Comment:
```suggestion
	hasReadme := bytes.Contains(dirres.Data, []byte("README.md\n"))
```

I think searching directly in the byte slice would be better then converting to a string and check for README.

This approach is better because:
- It reduces memory usage by avoiding string conversion.
- It searches directly without iterating through the entire file list.

Please note that, this method assumes that there's always a newline character (`\n`) at the end of each filename.

For reference, I've attached benchmark result below:

```bench
Benchmark_String-8   	   23318	     51086 ns/op	   81920 B/op	       2 allocs/op
Benchmark_Bytes-8   	 1000000	      1002 ns/op	       0 B/op	       0 allocs/op
```

Reply:
Thanks for the suggestion ! It appears that github files are sorted in ascii order so I think it's safe to assume there's always a `\n`.